benches/language.rs

```diff
@@ -0,0 +1,193 @@
+use criterion::{criterion_group, criterion_main, BatchSize, Criterion, SamplingMode, Throughput};
```

Comment:
Let's rename this file to `benches/languages.rs` for consistency with `make bench-languages`. It's a small nit, but I got bit by this, so I'm proposing it.